pandas/pandas_interoperability

```diff
@@ -0,0 +1,27 @@
+import Orange
+import pandas as pd
```

Comment:
This is a heavy import we'd rather not rely on. Is it possible to import it from within the function providing the functionality. As far as I can tell, importing it is not even needed here. :)